### DISCUSSION TITLE
How to get start with Pallas?

### DISCUSSION THREAD

**Discussion by demon2036** [jax-ml/jax#18316] [Q&A]
I would like to start with Pallas on my local GPU,but I find an error "NotImplementedError: MLIR translation rule for primitive 'pallas_call' not found for platform cuda" ,could anyone provide your last versions what worked for you?

**Comment by anh-tong** [comment]
I think this might be a duplication of #17367.

**Reply by demon2036** [reply]
Unfortunately,follow that guidance I still could not use Pallas on my local GPU.
